### PR TITLE
Examples: Clean up the spotlight example

### PR DESCRIPTION
**Pull request by WestLangley**
Clean up. The only major change is to reduce the light intensity. Granted, this is a subjective decision. Feel free to change it.

### CODE REVIEW

**Comment by Mugen87** [comment]
For better comparison:

https://threejs.org/examples/webgl_lights_spotlight
https://raw.githack.com/WestLangley/three.js/dev-spotlight_example/examples/webgl_lights_spotlight.html

File: examples/webgl_lights_spotlight.html [hunk 1]
```diff
@@ -95,8 +99,8 @@
 				spotLight.map = textures[ 'disturb.jpg' ];
 
 				spotLight.castShadow = true;
-				spotLight.shadow.mapSize.width = 1024;
-				spotLight.shadow.mapSize.height = 1024;
+				spotLight.shadow.mapSize.width = 512;
+				spotLight.shadow.mapSize.height = 512;
```
Comment (Mugen87) [inline examples/webgl_lights_spotlight.html:103]: When comparing the new and previous version, I notice that the edges of the statue's shadows flickering now more noticeably. 

Did you lower the resolution to soften the shadows?

Comment (WestLangley) [reply examples/webgl_lights_spotlight.html:103]: Oh, I didn't realize we used such large values in the examples.

I'll be glad to restore it.

Comment (WestLangley) [reply examples/webgl_lights_spotlight.html:103]: done!

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**